setup.cfg

```diff
@@ -7,8 +7,13 @@ cover-min-percentage=100
 cover-erase=true
 
 [flake8]
+exclude=./docs/conf.py,./test/*
```

Comment:
happy to also exclude dummyserver/, as it is mostly part of tests.  however it is quite clean, and could be re-used by other projects, so it is nice to provide clean code for other projects to copy and re-use.